docs/workloads/azure/data/getting_started/de_shared_resourcedeployment_azure.md

```diff
@@ -0,0 +1,64 @@
+title: DE shared resource Deployment
+sidebar_label: DE shared resource Deployment
+hide_title: false
+hide_table_of_contents: false
+description: DE shared resource deployment
+keywords:
+  - DE
+  - data
+  - infrastructure
+  - azure
+  - template
+---
+
+This section provides an overview of deploying Data Platform shared resources for Stacks Data Platform.
+This aligns to the workflow shown in the [deployment architecture](../architecture/architecture_data_azure.md#data-engineering-workloads) section.
+Data Plaform shared resources include the ADF resources which are shared in various ADF pipelines.
+
+It assumes all [prerequisites](../requirements_data_azure.md#azure) are in place, including:
+
+* Azure subscription and service principal
+* Azure DevOps project with [Pipelines variable groups](../requirements_data_azure.md#azure-pipelines-variable-groups)
+* A [deployed Stacks data platform](core_data_platform_deployment_azure.md)
+
+## Step 1: Create feature branch
+
+Open the project locally and create a new feature branch e.g.:
+
+```bash
+git checkout -b feat/de-shared-pipeline
+```
+The `de_build` folder includes YAML file called `job-pipeline-vars` that contains the variables used in the DE shared reource pipeline. These variables must be updated as per the project requirements.
+
+## Step 2: Add DE shared resource pipeline in Azure DevOps
+
+YAML file containing a template Azure DevOps CI/CD pipeline for building and deploying the DE shared reource, named `de-shared-resources.yml` is configured in the `de_workloads/shared_resources`
+
+YAML file `de-shared-resources.yml` should be added as the definition for a new pipeline in Azure DevOps.
+
+1. Sign-in to your Azure DevOps organization and go to your project
+2. Go to Pipelines, and then select **New pipeline**
+3. Name the new pipeline, e.g. `de-shared-resources`
+4. For the pipeline definition, specify the YAML file in the project repository feature branch (`de-shared-resources.yml`) and save
+5. The new pipeline will require access to any Azure DevOps pipeline variable groups specified in the pipeline YMAL. Under each variable group, go to 'Pipeline permissions' and add the pipeline.
+
+
+## Step 3: Deploy DE shared reource in non-production environment
+
+Run the pipeline configured in Step 2 to commence the build and deployment process.
+
+Running this pipeline in Azure DevOps will initiate the deployment of artifacts into the non-production (nonprod) environment. It's important to monitor the progress of this deployment to ensure its success. You can track the progress and status of the deployment within the Pipelines section of Azure DevOps.
+
+If successful, the core DE shared reource will now be available in the nonprod Stacks environment. To view these deployed resources, navigate to the [Azure portal](https://portal.azure.com/) and search for the resource group and ope ADF. Within the ADF, you'll find the shared resources that were deployed.
```

Comment:
Maybe some screenshot of these shared resources in ADF? I think we should somehow mention what exactly are those.

Reply:
added resource list